content/helm_root/helm_nginx/installnginx.md

```diff
@@ -8,11 +8,15 @@ Installing the Bitnami standalone NGINX web server Chart involves us using the [
 
 When we install using Helm, we need to provide a deployment name, or a random one will be assigned to the deployment automatically.
 
-Let's provide **mywebserver** as the deployment name as we install NGINX:
+Now, use the **helm** utility to **install** the **bitnami/nginx** chart and specify the name **mywebserver** for the Kubernetes deployment:
```

Comment:
I like the challenge question. Can you use format mentioned in Wiki for consistency? 
(https://github.com/aws-samples/eks-workshop/wiki/Workhop-testing)

Reply:
I updated with a second commit to fix the challenge formatting. can you take another look?

Reply:
it looks like this has been resolved.